migration/insitution/app.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const limiter = require('limiter');
+const readline = require('readline');
+const consentAPI = require('./consentAPI');
+const log = require('./logging');
+
+/**
+ * Simple script to post new Institutions given a file of institution names
+ *
+ * Requires authentication as an admin user.
+ *
+ * Sample Usage:
+ *   node app.js \
+ *     ./institutions.txt
+ *     `gcloud auth print-access-token` \
+ *     https://local.broadinstitute.org:27443 \
+ *
+ *
+ * EXAMPLE LINE IN FILE:
+ *  Polytechnique Fédérale de Lausanne
+ *
+ *  id, create user, create date, update user, and update date are automatically populated (even if given values they are ignored)
+ *  API call is in the form of:
+ *  POST $host/api/institutions -d {"name": "Polytechnique Fédérale de Lausanne"}
+ */
+
+// Program args in process.argv are last in the array. Pop them in reverse order:
+const host = process.argv.pop();
```

Comment:
Nit: Not a fan of reading directly from the argv array since it means there's a fixed argument order.

One alternative is to use a package called [yargs](https://github.com/yargs/yargs) and passing in the arguments as flags. This way you can just access them by name in the `argv` value provided by `yargs` and not worry about the order of the arguments.

Here's the example page for the library
https://github.com/yargs/yargs/blob/main/docs/examples.md

Reply:
Thank you, will look at that. I had read about another one, commander.js, but it seemed like overkill